code/modules/spells/artifacts.dm

```diff
@@ -142,8 +147,8 @@
 /obj/item/phylactery/Initialize()
 	. = ..()
 	world_phylactery += src
-	create_reagents(120)
-	reagents.add_reagent("undead_ichor", 120)
+	create_reagents(0)

```

Comment:
pretty sure this makes this literally useless. and wasn't mentioned in the PR at all. nice.